summary.qmd

```diff
@@ -4,35 +4,39 @@ title: "What is Open Source?"
 
 ## What is Open Source?
 
-When someone says 'open source' - what does that mean to you?\
-What are the important characteristics of something that is regarded as 'open source'?
+The [Open Source Initiative](https://opensource.org/osd){target="_blank"} delineates a number of characteristics of what it means to be **open source**, including free distribution, available source code, and the ability to create derived works.
```

Comment:
There's a "freedom" notion with Open Source - i.e. freedom to *use* as you wish and freedom to *modify* as you wish, within the constraints of the license and any code of conduct. It often means "free" as in at no cost, but that also often brings with it freedom from liability for the developer.